{"uuid": "48d0dd89-f432-4541-af74-807f3b280389", "befores": [{"name": "create_driver", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/tests/conftest.py\", line 61, in create_driver\n    driver = webdriver.Chrome(ChromeDriverManager().install(), options=opts)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/webdriver_manager/chrome.py\", line 32, in install\n    driver_path = self._get_driver_path(self.driver)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/webdriver_manager/manager.py\", line 22, in _get_driver_path\n    driver_version = driver.get_version()\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/webdriver_manager/driver.py\", line 40, in get_version\n    return self.get_latest_release_version()\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/webdriver_manager/driver.py\", line 64, in get_latest_release_version\n    validate_response(resp)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/webdriver_manager/utils.py\", line 81, in validate_response\n    raise ValueError(resp.json())\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/requests/models.py\", line 978, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1737705440845, "stop": 1737705441718}], "start": 1737705440845, "stop": 1737705441770}